packages/client/src/lib.js

```diff
@@ -13,11 +13,23 @@
  * ```
  * @module
  */
+
+// @ts-ignore module with no types
+import { parallelMap } from 'streaming-iterables'
+import pRetry from 'p-retry'
+
+import { CarReader } from '@ipld/car'
+import { TreewalkCarSplitter } from 'carbites'
+
 import * as API from './lib/interface.js'
 import * as Token from './token.js'
 import { fetch, File, Blob, FormData } from './platform.js'
 import { toGatewayURL } from './gateway.js'
 
+const MAX_STORE_RETRIES = 3
```

Comment:
We really want this to succeed...

```suggestion
const MAX_STORE_RETRIES = 5
```